probe/endpoint/reporter.go

```diff
@@ -14,6 +14,7 @@ import (
 
 // Node metadata keys.
 const (
+	Name = "name"
```

Comment:
I understand that having a _Name_ constant is not very nice, but I don't think having a structure like this:

```
			remoteNode = report.MakeNodeMetadataWith(map[string]string{
				"name": revRemoteAddr,
				Addr:   remoteAddr,
			})
```

where "name" is the only literal, does not make sense either, especially when some other parts of the code depend on that constant. Maybe we should standardise these names with something like:

 ```
			remoteNode = report.MakeNodeMetadataWith(map[string]string{
				NodeMetadataName: revRemoteAddr,
				NodeMetadataAddr: remoteAddr,
			})
```